### ISSUE TITLE
Update outdated links and add new resources 

### ISSUE THREAD

**Issue by theredditbandit** [theredditbandit/awesome-cybersec#10]
As the title suggests , this resource list hasn't been updated in a while and could do with a fresh coat of paint. Just to make sure the present links are working and add any new resources that might be there.

**Comment by SanskariWolf** [comment]
Can you please assign me this issue.

**Comment by theredditbandit** [comment]
Sure

**Comment by SanskariWolf** [comment]
Current Update:
I have checked/updated the links for the Discord server, YouTube channels, miscellaneous links, resources, and paid training sections, and everything is working well. 
I will now move on to checking the links in the other sections as outlined in the issue.

**Comment by theredditbandit** [comment]
This is good to hear , once you've verified all the valid links do you think we can restructure the readme , sort of like [build-your-own-x](https://github.com/codecrafters-io/build-your-own-x) repo has done to their readme to make the content more accessible. At the moment I feel the content in this repo is laid out all over the place. The least we can do is add an index at the top that links to the subheadings below

**Comment by theredditbandit** [comment]
also , regarding the discord links 
![image](https://github.com/user-attachments/assets/e6b5bb56-3248-4ac0-96d6-d10769738d73)

Discord has this "feature" where by default the links expire after 7 days, so make sure the links you have collected don't have an expiration time. You can do this using the edit invite link button.

**Comment by SanskariWolf** [comment]
Regarding the Discord link, I've considered the 7-day expiration. 
Once I update and verify all the links, I'll focus on improving the positioning and styling of the links in the form of chart or index to enhance accessibility.

**Comment by theredditbandit** [comment]
Sounds good , thanks!

**Comment by theredditbandit** [comment]
hey @SanskariWolf do you have an update on this

**Comment by SanskariWolf** [comment]
Yeah, I am still over going through the link and would be completing all the changes and creating a merge request by 18th October.

**Comment by theredditbandit** [comment]
Hey @SanskariWolf! , do you have an update on these changes
